fdbclient/Tuple.cpp

```diff
@@ -99,6 +99,48 @@ Tuple Tuple::unpack(StringRef const& str, bool exclude_incomplete) {
 	return Tuple(str, exclude_incomplete);
 }
 
+std::string Tuple::tupleToString(const Tuple& tuple, bool isSingleton) {
```

Comment:
You shouldn't need the `isSingleton` argument, the length of the tuple can be determined with it's `size` function.